### PR TITLE
Adding ABCORS to CKAN

### PR DESCRIPTION
**Pull request by zentarul**
I hope I did the git thing correctly, crossing my fingers.

### CODE REVIEW

File: NetKAN/ABCORS.netkan [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "spec_version" : 1,
+    "identifier"   : "ABCORS",
+    "$kref"        : "#/ckan/github/zentarul/abcors",
+    "name"         : "ABookCase Orbital Reference System",
+    "abstract"     : "Hover the mouse on your orbit to display contextual information.",
+    "license"      : "MIT",
+    "$vref"        : "#/ckan/ksp-avc",
+    "resources": {
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/131706-1",
+        "github": "https://github.com/zentarul/abcors"
```
Comment (Olympic1) [inline NetKAN/ABCORS.netkan]: Change `github` to `repository`

Comment (zentarul) [inline NetKAN/ABCORS.netkan]: I used the gui, so I'm not sure what you are talking about, I'm in the IRC right now though if you could give me 5 minutes to help me.

Comment (zentarul) [inline NetKAN/ABCORS.netkan]: nevermind, I am stupid, you mean as in the github in the sourcefile

**Comment by politas** [comment]
Ref #3043

**Pull request closed without merging**